### ISSUE TITLE
Epoch 00013: early stopping

### ISSUE THREAD

**Issue by IslamAtef95** [saif-mahmud/self-attention-HAR#2]
Epoch 00013: early stopping

why this happens?

I ran for 75 epochs

**Comment by saif-mahmud** [comment]
Early stopping happens when the metric on the validation set is not improving for specific epochs (like in plateau).
Ref: https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/EarlyStopping

You can comment out this line to make sure it runs for your defined epochs under any condition.
https://github.com/saif-mahmud/self-attention-HAR/blob/737539eb38569799cb63c7add5fd98677d099dbe/utils/train.py#L26

And, you have to remove the `earlyStopping` callback from this line:
https://github.com/saif-mahmud/self-attention-HAR/blob/737539eb38569799cb63c7add5fd98677d099dbe/utils/train.py#L39
